dev/com.ibm.jbatch.container/resources/com/ibm/jbatch/container/internal/resources/JBatchMessages.nlsprops

```diff
@@ -303,3 +303,7 @@ display.unresolved.jsl=CWWKY0043I:  The unresolved JSL for this job is \n {0}.
 display.unresolved.jsl.explanation=The JSL used for the job is displayed before parameters and properties are resolved.
 display.unresolved.jsl.useraction=Ensure that the file name for the JSL is correct and verify that the source of the JSL contains the correct JSL.
 
+#{0} The invalid group name as specified in the server configuration
+JOB_GROUP_NAME_NOT_VALID=CWWKY042W: The specified operationGroup name {0} could not be resolved to a unique group name.
+JOB_GROUP_NAME_NOT_VALID.explanation=Attempt to access the job via group access may fail if attempting to use an invalid group name
```

Comment:
See suggestion:
Attempts to access the job using group access might fail, if attempts to use an invalid group name occur.